### PR TITLE
Packaging Mac: Use 'Applications' instead 'application'

### PR DESCRIPTION
**Pull request by praveenkumar**

### CODE REVIEW

File: packaging/darwin/Resources/conclusion.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
 <div align="left" style="font-family: Helvetica; padding-left: 10px;">
     <br/>
     <p style="color: #020202; font-size: 12px;">Thanks for installing CodeReady Containers!</p>
-    <p style="color: #020202; font-size: 12px;">You can now open CodeReady Containers from application</b>.</p>
+    <p style="color: #020202; font-size: 12px;">You can now open CodeReady Containers from the Applications</b>.</p>
```
Comment (anjannath) [inline packaging/darwin/Resources/conclusion.html:10]: maybe drop `the` and just `You can now open CodeReady Containers from Applications` 
cc @robin-owen

Comment (gbraad) [reply packaging/darwin/Resources/conclusion.html:10]: Is there an official name for this folder?

Comment (gbraad) [reply packaging/darwin/Resources/conclusion.html:10]: https://support.apple.com/guide/mac-help/folders-that-come-with-your-mac-mchlp1143/mac

`You can now open CodeReady Containers from the Applications folder.`

**Review by gbraad — CHANGES_REQUESTED** [review]
`You can now open CodeReady Containers from the Applications folder`

**Review by anjannath — APPROVED** [review]

**Pull request merged**